### ISSUE TITLE
Source code of the tensorflow version

### ISSUE THREAD

**Issue by wangzheng121** [palver7/CFLPytorch#2]
Thanks to the author for providing the source code for the pytorch implementation. Could you please provide the source code for the tensorflow version?

**Comment by palver7** [comment]
I do not have the source code for the tensorflow version. The only source code I know is the one provided by the authors of the CFL paper: https://github.com/cfernandezlab/CFL

**Comment by wangzheng121** [comment]
Yes, I did find this article as well. It gave me a headache because the author did not provide a training script. If you also don't have the tensorflow source code, I'm very curious how you wrote the TFCFL model. Or if you know of any way to help me get the tensorflow training script, I would be very grateful!
从 Windows 版邮件发送

发件人: Daniel Kurnia Suhendro
发送时间: 2021年9月27日 17:17
主题: Re: [palver7/CFLPytorch] Source code of the tensorflow version (#2)

I do not have the source code for the tensorflow version. The only source code I know is the one provided by the authors of the CFL paper, which has no training script last I checked.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by palver7** [comment]
I used a guide from huggingface (https://medium.com/huggingface/from-tensorflow-to-pytorch-265f40ef2a28) to convert TF version of the trained model and weights to PyTorch. This guide will not provide how to create a working training script, however.

**Comment by wangzheng121** [comment]
Oh, I see. Maybe I should try to write the training script by myself. Thanks for your help!! 

从 Windows 版邮件发送

发件人: Daniel Kurnia Suhendro
发送时间: 2021年9月27日 17:42
主题: Re: [palver7/CFLPytorch] Source code of the tensorflow version (#2)

I used a guide from huggingface (https://medium.com/huggingface/from-tensorflow-to-pytorch-265f40ef2a28) to convert TF version of the trained model and weights to PyTorch.
—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
